Neos.ContentGraph.DoctrineDbalAdapter/src/ContentGraphReadModelAdapter.php

```diff
@@ -32,10 +32,9 @@
 use Neos\EventStore\Model\Event\Version;
 
 /**
- * @internal only used inside the
- * @see ContentRepositoryReadModel
+ * @internal
  */
-final readonly class ContentRepositoryReadModelAdapter implements ContentRepositoryReadModelAdapterInterface
+final readonly class ContentGraphReadModelAdapter implements ContentGraphReadModelInterface
```

Comment:
Just wondering: Why did you rename this?
That read model knows all about workspaces etc. and those are not really part of the content graph, are they?

Reply:
They are?! We moved them to the contentgraph projection, IMHO because they are inherently part of it now?

Reply:
I think you missed this yesterday then, sebastian and me suggested that this should all be ContentGraph instead of content repository.